core/src/parking_lot.rs

```diff
@@ -51,7 +52,7 @@ cfg_if! {
         // See https://github.com/rust-lang/rust/blob/master/src/libstd/sys/wasm/time.rs
         type InstantType = DummyInstant;
     } else {
-        // Otherwise use `std::time::Instant`
+        // Otherwise use `instant::Instant`
         type InstantType = Instant;
     }
 }
```

Comment:
Is this hack still necessary now that we have support for Instant on WASM? I guess the default instant type is still used if neither the `stdweb` or `wasm-bindgen` features are enabled...

Reply:
I'm not sure, but shall check (give me an hour)

Reply:
It works without the hack :v:

GIF of a single threaded WASM countdown (counts down quickly, but only refreshes at 10 FPS):

![wasm_countdown](https://user-images.githubusercontent.com/2993230/82100643-4cd57900-975e-11ea-8079-f2c02efe0f43.gif)

Reply:
After thinking about it for a bit, I think it's better to keep the old `DummyInstant` hack for wasm. The code using it relies on the fact that calling `Instant::now` is reasonably fast, which may not be the case on wasm.

Reply:
Gotcha, shall take take commit out